features/waarden.feature

```diff
@@ -189,28 +335,224 @@ Feature: leveren van waarden bij een WOZ-object
+        | hoge_raad_veranderd | 32 | arrest Hoge Raad, vastgestelde waarde veranderd |
+        | hoge_raad_gedingverwezen | 33 | arrestHogeRaad,gedingverwezen |
+        | voorlopige_aanslag | 99 | waarde te gebruiken voor voorlopige aanslag |
+
+    Scenario: indicatieBezwaarBeroep wanneer er meerdere statussen zijn bij een waardepeildatum en ingangsdatum
+      Gegeven WOZ-object met objectnummer "23280647970000" heeft de volgende beschikkingen:
+      | Peildatum  | WOZ-waarde   | Ingangsdatum | Datum beschikking | Status |
+      | 01-01-2019 | 437.000 euro | 01-01-2020   | 06-12-2020        | 21     |
+      | 01-01-2019 | 437.000 euro | 01-01-2020   | 13-01-2020        | 20     |
+      | 01-01-2018 | 392.000 euro | 01-01-2019   | 14-01-2019        | 01     |
+      | 01-01-2018 | 392.000 euro | 01-01-2019   | 14-01-2019        | 23     |
+      Als het WOZ object wordt opgevraagd met /wozobjecten/23280647970000
+      Dan levert de API de volgende waarden:
+      """
+      [
+        {
+          "vastgesteldeWaarde": 437000,
+          "waardepeildatum": "2019-01-01",
+          "ingangsdatum": "2020-01-01",
+          "beschikkingsStatussen": [ "beroep_gehandhaafd", "beroepaangetekend" ]
+        },{
+          "vastgesteldeWaarde": 392000,
+          "waardepeildatum": "2018-01-01",
+          "ingangsdatum": "2019-01-01",
+          "beschikkingsStatussen": [ "hogerberoepaangetekend", "beschikking_genomen" ]
+        }
+      ]
+      """
+      En zijn de daaruit bepaalde relevante de waarden:
+      """
+      "waarden": [
+        {
+          "vastgesteldeWaarde": 437000,
+          "waardepeildatum": "2019-01-01",
+          "indicatieBezwaarBeroep": true
+        },
+        {
+          "vastgesteldeWaarde": 392000,
+          "waardepeildatum": "2018-01-01",
+          "indicatieBezwaarBeroep": true
+        }
+      ]
+      """
+
+    Scenario: indicatieBezwaarBeroep wanneer er bezwaar, (hoger)beroep of cassatie loopt bij niet de meest recente ingangsdatum
+      Gegeven WOZ-object met objectnummer "22310827210003" heeft met ingang van 01-05-2019 een nieuwe eigenaar
+      En de gemeente heeft een beschikking gestuurd naar de nieuwe eigenaar over waardepeildatum 01-01-2018 met ingangsdatum 01-05-2019
+      En de vorige eigenaar heeft beroep ingediend over de WOZ-waarde op waardepeildatum 01-01-2018
+      En WOZ-object met objectnummer "22310827210003" heeft de volgende beschikkingen:
+      | Peildatum  | WOZ-waarde   | Ingangsdatum | Datum beschikking | Status |
+      | 01-01-2018 | 215.000 euro | 01-01-2019   | 28-05-2019        | 01     |
+      | 01-01-2018 | 215.000 euro | 01-05-2019   | 14-01-2019        | 20     |
+      Als het WOZ object wordt opgevraagd met /wozobjecten/22310827210003
+      Dan levert de API de volgende waarden:
+      """
+      [
+        {
+          "vastgesteldeWaarde": 215000,
+          "waardepeildatum": "2018-01-01",
+          "ingangsdatum": "2018-05-01",
```

Comment:
2018-05-01 moet volgens mij 2019-01-01 zijn

Reply:
moest zijn 2019-05-01, maar in de tabel was ook van alles gehusseld, heb ik nu ook gecorrigeerd